src/governance.spec.ts

```diff
@@ -204,7 +204,10 @@ describe("GovernanceCanister", () => {
         stake: ICP.fromString("1") as ICP,
         principal: new AnonymousIdentity().getPrincipal(),
         ledgerCanister: mockLedger,
-        fromSubAccountId: 1234,
+        fromSubAccount: [
```

Comment:
should we add a test for the case `fromSubaccount = undefined`?

Reply:
It's above, in the test "creates new neuron successfully", that one I didn't change.